gradle.properties

```diff
@@ -1,2 +1,10 @@
 # Uncomment to build for Android
-#target = android
\ No newline at end of file
+#target = android
+
+# See README.md 'How to release to maven central'
+signing.keyId={key_id}
+signing.password={key_passphrase}
+signing.secretKeyRingFile={secret_keys_ring_file}
+
+ossrhUsername={username}
+ossrhPassword={password}
```

Comment:
ohhh, variables

Reply:
Yeah, the prick is to use a local version gradle.properties. I have provided instructions in the readme.